apps/studio/components/interfaces/SQLEditor/UtilityPanel/Results.tsx

```diff
@@ -46,6 +46,7 @@ const Results = ({ id, rows }: { id: string; rows: readonly any[] }) => {
     return (
       <span
         className="font-mono text-xs w-full"
```

Comment:
```suggestion
        className="font-mono text-xs w-full whitespace-pre"
```
--


<br class="Apple-interchange-newline">